src/Interpreters/SystemLog.h

```diff
@@ -5,6 +5,32 @@
 #include <Parsers/IAST.h>
 
 #include <boost/noncopyable.hpp>
+#include <vector>
+
+#define LIST_OF_ALL_SYSTEM_LOGS(M) \
+    M(QueryLog, query_log, "Used to log queries.") \
```

Comment:
FYI We have comments here: https://github.com/ClickHouse/ClickHouse/blob/4cdb6db1b7bb212d655cde15c0eb8534687d6c9d/src/Interpreters/SystemLog.cpp#L285-L312

Maybe we can just reuse them?

Reply:
Actually I do not need the description in LIST_OF_ALL_SYSTEM_LOGS. I just wanted to save comments from old code. 
But you are right, no need to do it twice, I will change it.